### ISSUE TITLE
PhoneLogs.numberOfCalls - behaviour when no folderID is specified

### ISSUE THREAD

**Issue by brentlintner** [blackberry/WebWorks-API-Docs#237]
The docs lists the signature for numbeOfCalls as:

```
static Number numberOfCalls(folderID : Number)
```

where folderID is on of the FOLDER_\* values (this is the same for some other phoneLogs apis as well).

However when no folderID is specified, what is the default behaviour? Currently it looks like the folderID is set to -1 in the native source [here](https://github.com/blackberry/WebWorks/blob/master/api/phone/src/main/java/blackberry/phone/calllog/CallLogNamespace.java#L144-146), before being sent to a native [phoneLogs.numberOfCalls](http://www.blackberry.com/developers/docs/6.0.0api/net/rim/blackberry/api/phone/phonelogs/PhoneLogs.html#numberOfCalls(long) call.

I could not find any specific documentation on it, but if no one knows off the top of their head I will boot up a webworks app and and see what is there. :-)

Thanks!

**Comment by jeffheifetz** [comment]
Did you ever find the answer Brent?

**Comment by brentlintner** [comment]
I don't believe I did. Feel free to close this though (so long ago I definitely forgot about this).
